tests/devices/test_default_clifford.py

```diff
@@ -0,0 +1,356 @@
+    """Test that the execution of multiple tapes is possible with multiprocessing on this device."""
+
+    os.environ["OMP_NUM_THREADS"] = "4"
+
+    dev_c = qml.device("default.clifford", max_workers=2)
+    dev_q = qml.device("default.qubit", max_workers=2)
+
+    qscript = qml.tape.QuantumScript(
+        [qml.Hadamard(wires=[0]), qml.CNOT(wires=[0, 1])],
+        [qml.expval(qml.PauliX(0))],
+    )
+    tapes = (qscript, qscript)
+
+    program, conf_d = dev_c.preprocess()
+    tapes_clrfd, _ = program(tapes)
+    res_c = dev_c.execute(tapes, conf_d)
+    program, conf_q = dev_q.preprocess()
+    tapes_qubit, _ = program(tapes)
+    res_q = dev_q.execute(tapes, conf_q)
+    assert np.allclose(res_q, res_c)
+
+    grad_c = dev_c.compute_derivatives(tapes_clrfd, conf_q)
+    grad_q = dev_q.compute_derivatives(tapes_qubit, conf_d)
+    assert qml.math.allclose(grad_q, grad_c)
+
+    (res_c, grad_c) = dev_c.execute_and_compute_derivatives(tapes_clrfd, conf_q)
+    assert qml.math.allclose(res_q, res_c) and qml.math.allclose(grad_q, grad_c)
+
+
+def test_tracker():
+    """Test that the tracker works for this device."""
+
+    dev_c = qml.device("default.clifford")
+    dev_q = qml.device("default.qubit")
+
+    qscript = qml.tape.QuantumScript(
+        [qml.Hadamard(wires=[0]), qml.CNOT([0, 1])],
+        [qml.expval(qml.PauliZ(0))],
+    )
+    tapes = tuple([qscript])
+
+    with qml.Tracker(dev_c) as tracker:
+        program, conf_d = dev_c.preprocess()
+        tapes_clrfd, _ = program(tapes)
+        res_c = dev_c.execute(tapes, conf_d)
+        program, conf_q = dev_q.preprocess()
+        tapes_qubit, _ = program(tapes)
+        res_q = dev_q.execute(tapes, conf_q)
+        assert np.allclose(res_q, res_c)
+
+        res_s = dev_c.execute(qscript, conf_d)
+        assert np.allclose(res_c, res_s)
+
+        grad_c = dev_c.compute_derivatives(tapes_clrfd, conf_q)
+        grad_q = dev_q.compute_derivatives(tapes_qubit, conf_d)
+        assert qml.math.allclose(grad_q, grad_c)
+
+        (res_c, grad_c) = dev_c.execute_and_compute_derivatives(tapes_clrfd, conf_q)
+        assert qml.math.allclose(res_q, res_c) and qml.math.allclose(grad_q, grad_c)
+
+    assert tracker.totals == {
+        "batches": 3,
+        "simulations": 3,
+        "executions": 4,
+        "derivative_batches": 2,
+        "derivatives": 3,
+        "execute_and_derivative_batches": 1,
+    }
+    assert np.allclose(tracker.history.pop("results")[0], 0.0)
+    assert tracker.history.pop("resources")[0] == qml.resource.Resources(
+        num_wires=2,
+        num_gates=2,
+        gate_types={"Hadamard": 1, "CNOT": 1},
+        gate_sizes={1: 1, 2: 1},
+        depth=2,
+    )
+    assert tracker.history == {
+        "batches": [1, 1, 1],
+        "simulations": [1, 1, 1],
+        "executions": [1, 1, 1, 1],
+        "derivative_batches": [1, 1],
+        "derivatives": [1, 1, 1],
+        "execute_and_derivative_batches": [1],
+    }
+
+
+def test_debugger():
```

Comment:
Might be worth adding an integration test with `qml.snapshots`.

Reply:
Added it!